HPXMLtoOpenStudio/tests/test_hvac.rb

```diff
@@ -803,6 +803,35 @@ def test_install_quality_mini_split_heat_pump_ratio
     _check_install_quality_multispeed_ratio(heat_pump, model, heat_pump)
   end
 
+  def test_custom_seasons
+    args_hash = {}
+    args_hash['hpxml_path'] = File.absolute_path(File.join(sample_files_dir, 'base-hvac-seasons.xml'))
+    model, hpxml = _test_measure(args_hash)
+
+    # Get HPXML values
+    hvac_control = hpxml.hvac_controls[0]
+    seasons_heating_begin_month = hvac_control.seasons_heating_begin_month
+    seasons_heating_begin_day = hvac_control.seasons_heating_begin_day
+    seasons_heating_end_month = hvac_control.seasons_heating_end_month
+    seasons_heating_end_day = hvac_control.seasons_heating_end_day
+    seasons_cooling_begin_month = hvac_control.seasons_cooling_begin_month
+    seasons_cooling_begin_day = hvac_control.seasons_cooling_begin_day
+    seasons_cooling_end_month = hvac_control.seasons_cooling_end_month
+    seasons_cooling_end_day = hvac_control.seasons_cooling_end_day
+
+    # Get zone
+    unitary_system = model.getAirLoopHVACUnitarySystems[0]
+    zone = unitary_system.controllingZoneorThermostatLocation.get
+
+    # Check heating season
+    heating_days = zone.sequentialHeatingFractionSchedule(zone.airLoopHVACTerminals[0]).get.to_ScheduleRuleset.get
+    assert_equal(heating_days.scheduleRules.size, 3)
+
+    # Check cooling season
+    cooling_days = zone.sequentialCoolingFractionSchedule(zone.airLoopHVACTerminals[0]).get.to_ScheduleRuleset.get
+    assert_equal(cooling_days.scheduleRules.size, 3)
```

Comment:
These tests don't check very much. Can you check that the schedule rule dates align with the season dates and that the appropriate value (zero or one) is specified?